File: app/src/main/java/com/kota/ASFramework/PageController/ASLinearLayout.java
package com.kota.ASFramework.PageController;

import android.content.Context;
import android.util.AttributeSet;
import android.widget.LinearLayout;

/* loaded from: classes.dex */
public class ASLinearLayout extends LinearLayout {
  public ASLinearLayout(Context context, AttributeSet attrs) {
    super(context, attrs);
  }

  public ASLinearLayout(Context context) {
    super(context);
  }

  public void notifyDataSetChanged() {
  }

  @Override // android.view.View, android.view.ViewParent
  public void requestLayout() {
    super.requestLayout();
  }
}

File: app/src/main/java/com/kota/asFramework/pageController/ASLinearLayout.kt
package com.kota.asFramework.pageController

import android.content.Context
import android.util.AttributeSet
import android.widget.LinearLayout

class ASLinearLayout : LinearLayout {
    constructor(context: Context?, attrs: AttributeSet?) : super(context, attrs)

    constructor(context: Context?) : super(context)

    fun notifyDataSetChanged() {
    }

    // android.view.View, android.view.ViewParent
    override fun requestLayout() {
        super.requestLayout()
    }
}
